docs/source/modin/supported/general_supported.rst

```diff
@@ -25,6 +25,9 @@ Data manipulations
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``cut``                     | P                               | ``retbins``, ``labels``          | ``N`` if ``retbins=True``or ``labels!=False``      |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
+| ``__dataframe__``           | P                               |                                  | ``N`` for columns of type ``Timedelta`` and columns|
+|                             |                                 |                                  | containing list objects                            |
```

Comment:
what does N mean here, in general N will throw a NotImplementedError, is that the same here?